src/HttpRequest.h

```diff
@@ -254,5 +254,9 @@ class ConnStateData;
  */
 void UpdateRequestNotes(ConnStateData *csd, HttpRequest &request, NotePairs const &notes);
 
+/// \returns a pointer to known *_port address where the request was accepted
+/// \returns nil for nil requests and when the port address info was not available
```

Comment:
With doxygen syntax we only get to use one \returns statement. If you need to document specific return values use \retval which takes two parameters with whitespace separation - the value and a description string for that values output case.

Consider:

 /**
  * The listening IP address from squid.conf *_port which the transaction client was connected to.
  *  May be [::] or 0.0.0.0.
  *
  * \retval *           object with IP address details.
  * \retval nullptr   no IP was found.
 **/

Reply:
Done